charts/kyverno/values.yaml

```diff
@@ -31,7 +31,7 @@ testImage:
   # testImage.repository defaults to "busybox" if omitted
   repository:
   # testImage.tag defaults to "latest" if omitted
-  tag:
+  tagTest:
```

Comment:
@realshuting was this an expected change? I can't see `tagTest` used anywhere?

Reply:
Hmmm, we might have to update the sed to handle this correctly. tagTest is not used anywhere in chart, the chart still references `tag` at https://github.com/kyverno/kyverno/blob/326d141a6d687d1a9839a8402e379f20e242c908/charts/kyverno/templates/tests/test.yaml#L21

Reply:
Right.. Logged https://github.com/kyverno/kyverno/issues/3172 to track.